specs/i18n.spec.js

```diff
@@ -122,5 +123,25 @@ describe('i18n', () => {
         done()
       }).catch(done)
     })
+
+    it('should consider the msgctxt option, if provided', (done) => {
+        setLocale('de_DE')
+
+        init(getLangLoader, config).then(() => {
+          expect(i18n('Export', {}, {msgctxt: 'button label'})).to.equal('Exportieren')
```

Comment:
Even though it works, this is not really how `i18n` is supposed to be called. It should only have 3 parameters when using pluralization, otherwise the options are passed as second argument.

Interpolations and options are thrown together into one object. This means there are some reserved options keys, which cannot be used as interpolation keys (so far only `markdown`).